lib/node_modules/@stdlib/stats/base/dvarmpn/examples/c/example.c

```diff
@@ -17,21 +17,20 @@
 */
 
 #include "stdlib/stats/base/dvarmpn.h"
-#include <stdint.h>
 #include <stdio.h>
 
 int main( void ) {
 	// Create a strided array:
-	double x[] = { 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 };
+	const double x[] = { 1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f };
```

Comment:
```suggestion
	const double x[] = { 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 };
```
only single precision floating-point values are suffixed with a `f`